index.js

```diff
@@ -81,8 +81,16 @@ async function watchRelativeDeps() {
 async function libraryHasChanged(name, libDir, targetDir, hashStore) {
   const hashFile = path.join(targetDir, "node_modules", name, ".relative-deps-hash")
   const referenceContents = fs.existsSync(hashFile) ? fs.readFileSync(hashFile, "utf8") : ""
-  // compute the hahses
-  const libFiles = await findFiles(libDir, targetDir)
+  // glob pattern list, relative to libDir, made for adding "dist" and similar
+  relativeDepsIgnoreFile = path.join(libDir, ".relative-deps-ignore")
+  const additionalIgnoredList = fs.existsSync(relativeDepsIgnoreFile)
+    ? fs.readFileSync(relativeDepsIgnoreFile, "utf8")
+      .split("\n")
+      .filter(line => line.trim() && !line.trim().startsWith('#'))
+      .map(line => line.split('#')[0])
```

Comment:
So in addition to checking that the file exists, there may be other causes of errors in reading the file contents (such as a lack of permissions, or the file may be huge and fail to be read into memory or a number of other issues).
To mitigate that, at the very least this block needs to be wrapped in try/catch. But instead of trying to mitigate all possible issues here, I'm sure there's a package which handles reading/parsing `.gitignore` format instead.